src/app/js/formats/shared/LodexResource.js

```diff
@@ -35,22 +35,46 @@ const styles = StyleSheet.create({
 });
 
 // see https://jsonfeed.org/version/1#items
-const LodexResource = ({ id, title, summary }) => {
+const LodexResource = ({ id, url, title, summary }) => {
     if (!id) {
         return null;
     }
-    return (
-        <div id={id}>
-            <Link
-                className={css(styles.contentLink)}
-                to={getFullResourceUri({ uri: id })}
-            >
+    let urlDomain = url.replace(id, '');
+    let getFullResourceUriDomain = getFullResourceUri({ uri: id }).replace(
+        id,
+        '',
+    );
+
+    const content = () => {
+        return (
+            <div>
                 <div className={css(styles.contentTitle)}>{title}</div>
                 <div className={css(styles.contentParagraph)}>{summary}</div>
                 <div className={css(styles.contentCustomDiv)} />
-            </Link>
-        </div>
-    );
+            </div>
+        );
+    };
+
+    if (urlDomain == getFullResourceUriDomain) {
```

Comment:
Maybe clearer using `if (!isLocalURL(id)) {`
(isLocalURL already exists in [src/common/uris.js](https://github.com/Inist-CNRS/lodex/blob/master/src/common/uris.js#L8-L13))

What do you think?